MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -207,9 +215,11 @@ private static IEnumerable<Directory> ProcessChunk([NotNull] PngChunk chunk)
             else if (chunkType == PngChunkType.tEXt)
             {
                 var reader = new SequentialByteArrayReader(bytes);
-                var keyword = reader.GetNullTerminatedString(maxLengthBytes: 79);
+                var keyword = reader.GetNullTerminatedStringValue(maxLengthBytes: 79).ToString(defaultEncoding);
```

Comment:
It'd be nice to pass the encoding into `GetNullTerminatedStringValue`, so that it is paired with the bytes. After, just call ToString. Doesn't matter so much for this anonymous temporary, but in general I think that's a nicer API. It makes you think about the encoding when you're reading the string out.